packages/docusaurus-theme-classic/src/theme/DocSidebar/index.tsx

```diff
@@ -64,6 +65,7 @@ function DocSidebarItemCategory({
     }
     return isActive ? false : item.collapsed;
   });
+  const [initialLink, setInitialLink] = useState('');
```

Comment:
Let's avoid using empty string here and elsewhere in this PR. Use `null` or `undefined` instead.

Reply:
Commit b8e9f035 changes from 'empty string to undefined.